### DISCUSSION TITLE
Provisioning a new project is taking many hours

### DISCUSSION THREAD

**Discussion by barryburd** [supabase/supabase#30633] [Questions] [answered]
I'm on the free plan because I'm writing code to teach college students how to connect to a database. After creating a second project, Supabase tells me that I can't create a third project in the free plan. OK. So, I deleted my second project and tried to create another (second) project. Supabase lets me try to create a second project, but the project provisioning is still going after more than twelve hours. How to handle this?

**Comment by GaryAustin1** [comment, marked as answer]
If your project is stuck and you have hard refreshed the browser, or tried a different browser, then you need to contact support.
